packages/onboarding/src/setup-tailored-site-after-creation.ts

```diff
@@ -75,7 +74,8 @@ export function setupSiteAfterCreation( { siteId, flowName }: SetupOnboardingSit
 		if ( isNewsletterOrLinkInBioFlow( flowName ) || isFreeFlow( flowName ) ) {
 			// link-in-bio and link-in-bio-tld are considered the same intent.
 			if ( isLinkInBioFlow( flowName ) || isFreeFlow( flowName ) ) {
-				settings.site_intent = isLinkInBioFlow( flowName ) ? LINK_IN_BIO_FLOW : FREE_FLOW;
+				// We removed the link in bio flow, we need to keep this reference here to avoid side effects.
```

Comment:
because we need for link-in-bio-tld I assume

Reply:
Yup, I could change what we set as intent, but there might be a plethora of side effects.